localstack/services/s3/s3_listener.py

```diff
@@ -476,7 +476,7 @@ def fix_location_constraint(response):
         content = to_str(response.content or '') or ''
     except Exception:
         content = ''
-    if 'LocationConstraint' in content:
+    if aws_stack.get_region() != 'us-east-1' and 'LocationConstraint' in content:
```

Comment:
nitpick: We could assign the value of `aws_stack.get_region()` to a variable, instead of calling it twice (to optimize performance a bit).

Reply:
Sure. Will update it in follow up PR.